### ISSUE TITLE
Broken multiple hashtag

### ISSUE THREAD

**Issue by e-jigsaw** [progfay/scrapbox-parser#3]
**Describe the bug**
Parse broken when multiple hashtag in one line.

**To Reproduce**
https://runkit.com/embed/p7v8qvex4cmd

**Expected behavior**
```
0: Object {href: "foo", type: "hashTag"}
1: Object {text: " ", type: "plain"}
2: Object {href: "bar", type: "hashTag"}
3: Object {text: " ", type: "plain"}
4: Object {text: "hoge", type: "hashTag"}
```

**Screenshots**
[![Screenshot from Gyazo](https://gyazo.com/9b9dcb397e871f7a3b7e88a8f9821073/raw)](https://gyazo.com/9b9dcb397e871f7a3b7e88a8f9821073)

**Desktop (please complete the following information):**
 - OS: macOS 10.14.6
 - Browser Node.js
 - Version 10.16.3

**Comment by e-jigsaw** [comment]
@progfay ping

**Comment by progfay** [comment]
Thanks!
I fixed it, please check #5 .
